examples/07_joining_tables_with_FuzzyJoin.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/dirty-cat/datasets/master/data/Happiness_report_2022.csv",
+    thousands=",",
+)
+df.drop(df.tail(1).index, inplace=True)
+
+# Let's take a look at the table:
+df.head(3)
+
+#################################################################
+# The Happiness score was computed using the Gallup World Poll survey results.
+# The report stress out some of the possible explanatory factors: GDP per capita, Social support, Generosity etc.
+# However, these factors here are only estimated indexes used to calculate the happiness score.
+# Thus, we will not use them for our prediction model.
+###############################################################################
+# The sum of all explanatory indexes is then the happiness score itself:
+df["Sum_of_factors"] = df.iloc[:, [5, 6, 7, 8, 9, 10, 11]].sum(axis=1)
+df[["Happiness score", "Sum_of_factors"]].head(3)
+#################################################################
+X = df[["Country"]]
+y = df[["Happiness score"]]
+# We defined our X and y variables.
+###############################################################################
+# If we want to create a machine learning model which predicts
+# the happiness index of any new country or future date,
+# we will need to include explanatory factors from other tables.
+
+###############################################################################
+# Finding additional tables
+# ---------------------------
+#
+# Let's inspire ourselfes from the factors used by the Happiness report to explain happiness.
+# We will extract data from the World Bank databank using the following function:
+from dirty_cat.datasets import fetch_world_bank_indicator
+
+#################################################################
+# We then extract GDP per capita by country:
+gdppc = fetch_world_bank_indicator(
+    indicator_id="NY.GDP.PCAP.CD", indicator_name="gdppc"
+).X
+gdppc.head(3)
+
+#################################################################
+# Life expectancy by country:
+life_exp = fetch_world_bank_indicator("SP.DYN.LE00.IN", "life_exp").X
+life_exp.head(3)
+
+#################################################################
+# And the legal rights strength by country:
+legal_rights = fetch_world_bank_indicator("IC.LGL.CRED.XQ", "legal_rights").X
+legal_rights.head(3)
+
+###############################################################################
+# Joining World Bank tables to our initial one
+# ----------------------------------------------
+#
+# Now, using dirty_cat's :func:`fuzzy_join` function,
+# we need only one line to join two tables
+# without worrying about preprocessing:
+#
+# We add GDP per capita to the initial table:
+from dirty_cat.experimental import enable_fuzzy_join
+from dirty_cat import fuzzy_join
+
+X1, dist1 = fuzzy_join(X, gdppc, on=["Country", "Country Name"], return_distance=True)
+X1.head(20)
+# .. topic:: Note:
```

Comment:
```suggestion

#################################################################
# .. topic:: Note:
```

As you can see from generate page, the documentation is not generate right if you don't put this